src/pages/studyView/StudyViewPage.tsx

```diff
@@ -108,6 +109,10 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
         );
     }
 
+    private get appStore(){
```

Comment:
you can now get the appStore at the page level by using @injection.   see the ResultsViewPage